### ISSUE TITLE
Enable showing code block markers

### ISSUE THREAD

**Issue by aviyag** [xolox/vim-notes#107]
I want to see start-end markers of embedded code with syntax-highlighting, and see also the "type".
So, I have done the following change:
~/.vim/autoload/xolox/notes.vim line 1184

```vim
 execute printf(command, group, startgroup, startmarker, ft, endgroup, endmarker, include, ( has('conceal') && xolox#misc#option#get('notes_conceal_code', 1) )  ? ' concealends' : '')
```
Added checking for g:notes_conceal_code

Avi

**Comment by xolox** [comment]
Hi Avi and thanks for the suggestion! I was wondering when someone would follow up to #104 with the request to make the concealing of the  markers for code blocks configurable as well. Now let's see how long it takes for someone to request separate options for concealing inline vs. block markers :-).

**Comment by xolox** [comment]
I was a bit too quick posting my previous message, what I really wanted to say: This is now implemented as you suggested in vim-notes version 0.33. I updated the documentation as well.
